app/src/ui/window/full-screen-info.tsx

```diff
@@ -154,7 +153,8 @@ export class FullScreenInfo extends React.Component<
         timeout={toastTransitionTimeout}
       >
         <div key="notification" className="toast-notification">
-          Press <kbd>{kbdShortcut}</kbd> to exit fullscreen
+          Press <KeyboardShortcut darwinKeys={['⌘', 'F']} keys={['F11']} /> to
+          exit fullscreen
```

Comment:
One key is missing for macOS:

```suggestion
          Press <KeyboardShortcut darwinKeys={['^', '⌘', 'F']} keys={['F11']} />{' '}
          to exit fullscreen
```